pkg/pillar/cmd/zedagent/handlecertconfig.go

```diff
@@ -341,6 +341,7 @@ func sendAttestReqProtobuf(attestReq *attest.ZAttestReq, iteration int) {
 	deferKey := "attest:" + zcdevUUID.String()
 	zedcloud.RemoveDeferred(zedcloudCtx, deferKey)
```

Comment:
Even though attestation mssages are priority it makes sense to use the same pattern with SetDeferred + HandleDeferred here

Reply:
Done